src/views/token-swap/components/amount-box/amount-box.view.jsx

```diff
@@ -22,38 +22,41 @@ function AmountBox ({
   preferredCurrency,
   fiatExchangeRates,
   value,
+  amount,
   position,
   accounts,
   isDropdownActive,
   onInputChange,
   onTokenChange,
-  onActiveDropdownChange
+  onActiveDropdownChange,
+  onPositionUpdated,
```

Comment:
Last word of the handler name should be always in present:`onPositionUpdate` instead of `onPositionUpdated`.